WordPress/src/main/java/org/wordpress/android/push/GCMMessageService.java

```diff
@@ -699,9 +699,10 @@ private void showNotificationForBuilder(NotificationCompat.Builder builder, Cont
                 if (notifyUser) {
                     boolean shouldVibrate = prefs.getBoolean(context.getString(R.string.wp_pref_notification_vibrate), false);
                     boolean shouldBlinkLight = prefs.getBoolean(context.getString(R.string.wp_pref_notification_light), true);
-                    String notificationSound = prefs.getString(context.getString(R.string.wp_pref_custom_notification_sound), "content://settings/system/notification_sound"); //"" if None is selected
+                    String notificationSound = prefs.getString(context.getString(R.string.wp_pref_custom_notification_sound),
+                            context.getString(R.string.notification_settings_item_sights_and_sounds_choose_sound_default));
 
-                    if (!TextUtils.isEmpty(notificationSound)) {
+                    if (!TextUtils.isEmpty(notificationSound) && !notificationSound.trim().startsWith("file://")) {
```

Comment:
same comment as below, about making the comparison case-insensitive

Reply:
Done with 00363c0.